core/js/src/main/scala/plotly/element/PlotlyJavaTimeConversions.scala

```diff
@@ -0,0 +1,5 @@
+package plotly.element
+
+trait PlotlyJavaTimeConversions {
+  // Empty since the java.time classes are not available in ScalaJS
```

Comment:
Worth noting that another solution here would be to add a dependency on [`scala-java-time`](https://github.com/cquiroz/scala-java-time) from the core-js project. Let me know if you'd like to do this.

Reply:
I guess it's fine this way, let's not add an extra dependency…